tools/cuds-prioritized-attribution/composer/dependencies/helper_function.py

```diff
@@ -110,16 +110,16 @@ def local_to_gcs(bucket_name: str, object_name: str, input_file: str) -> None:
         blob.upload_from_file(file_obj)
 
 
-def convert_to_schema(commitment_schema: str) -> List[bigquery.SchemaField]:
+def convert_to_schema(schema: list) -> List[bigquery.SchemaField]:
```

Comment:
nit: we can make type hint stronger making code more self documenting and empowering IDE to catch more sooner.
```suggestion
def convert_to_schema(schema: List[Dict[str, str]]) -> List[bigquery.SchemaField]:
```

Reply:
done! thanks for the suggestion.